airtime_mvc/application/models/Preference.php

```diff
@@ -602,7 +607,14 @@ public static function SetStationLogo($imagePath)
 
     public static function GetStationLogo()
     {
-        return self::getValue("logoImage");
+        $logoImage = self::getValue("logoImage");
+        if (!empty($logoImage)) {
+            return $logoImage;
+        } else {
+            // We return the Airtime logo if no logo is set in the database.
+            // airtime_logo.png is stored under the public directory
+            return "airtime_logo.png";
```

Comment:
This is misleading because in airtime_mvc/application/views/scripts/form/preferences_general.phtml you're actually setting the image source to "airtime_logo.png" conditionally. (You've repeated yourself here...)

By reading the code above, I would expect that changing the "airtime_logo.png" string to "foobar.png" would make the default logo point to "foobar.png", but the code in preferences_general.phtml doesn't work like that.

I didn't realize that this function had to return a base64-encoded image. In this case, I think it'd be better to return a magic constant indicating that the default station logo should be used, and then also set a constant for that. (See my comments below in preferences_general.phtml for clarification).

Specifically, on line 616 above I'd return DEFAULT_LOGO_PLACEHOLDER, and define that in constants.php.